qgis_color_func.py

```diff
@@ -0,0 +1,72 @@
+import os
+
+
+def conversion(table_path, output_path):
+    # Create the directories if they do not exist
+    os.makedirs(os.path.dirname(output_path), exist_ok=True)
+
+    color_dict = {
+        'aqua': "100 128 255",
+        'black': "0 0 0",
+        'blue': "0 0 255",
+        'brown': "180 77 25",
+        'cyan': "0 255 255",
+        'gray': "128 128 128",
+        'grey': "128 128 128",
+        'green': "0 255 0",
+        'indigo': "0 128 255",
+        'magenta': "255 0 255",
+        'orange': "255 128 0",
+        'red': "255 0 0",
+        'violet': "128 0 255",
+        'purple': "128 0 255",
+        'white': "255 255 255",
+        'yellow': "255 255 0",
+    }
+
+    # Open the table
+    with open(table_path, 'r') as table:
+        table_lines = table.readlines()
+
+    # replace color names with rgb values
+    for i, line in enumerate(table_lines):
+        for color in color_dict:
+            if color in line:
+                table_lines[i] = line.replace(color, color_dict[color])
+
+    # Create the output table
+    with open(output_path, 'w') as output_table:
+        qml_header = open("qml_form", "r")

```

Comment:
Template file should have some file extension for better undestanding, eg.

```suggestion
        qml_header = open("qml_form.template", "r")

```